player.cpp

```diff
@@ -0,0 +1,31 @@
+#include "player.h"
+
+Player::Player()
+{
+
+}
+
+void Player::intitialize(Ui::MainWindow *ui_ptr)
+{
+    ui = ui_ptr;
+}
+
+void Player::display_choice(Choices player_choice)
+{
+    switch (player_choice) {
+    case ROCK:
+        ui->player1Label->setText("ROCK");
+        break;
+    case PAPER:
+        ui->player1Label->setText("PAPER");
+        break;
+    case SCISSORS:
+        ui->player1Label->setText("SCISSORS");
```

Comment:
These switch case is being used a lot in our code, I wonder if we can use unordered_map to reduce using switch case

Reply:
We can use that. I'll update it in the next commit.

Reply:
done